config-model/src/main/resources/schema/content.rnc

```diff
@@ -82,7 +82,7 @@ ClusterControllerTuning = element cluster-controller {
     element stable-state-period {  xsd:string { pattern = "([0-9\.]+)\s*([a-z]+)?" }  }? &
     element min-distributor-up-ratio { xsd:double }? &
     element min-storage-up-ratio { xsd:double }? &
-    element max-groups-allowed-down { xsd:nonNegativeInteger }?
+    element min-group-up-ratio { xsd:double }?
```

Comment:
Hm, I checked with Henning:  Both he and I intuitively preferred the negated field, i.e. something like `groups-allowed-down-ratio`.  Henning preferred to leave out a `max-` prefix, and that seems slightly better to me too.

Reply:
But now we see min-group-up-ratio matches the two other min-X-up-ratio names :thinking: .  The semantics of those may be sufficiently different that it confuse more than help?   I think the current min-X-up-ratio fields cause the cluster to go down, and is unrelated to orchestration?

Reply:
Naming was done in same style as the other `min-X-up-ratio`, but I agree that `groups-allowed-down-ratio` is easier to understand and work with, I will change the name